### ISSUE TITLE
Why don't have SelectAllCompleteAction of countryReducer

### ISSUE THREAD

**Issue by JeffreyChan** [damienbod/AngularNgrx#1]
Hi Damien
  I don't understand if you don't have select_all_complete_action of reducer, why do you dispatch it?

  It seems to me waste http invoke resources?

**Comment by damienbod** [comment]
@JeffreyChan yes, my mistake, I removed the country select ALL, but did not clean it up properly. I removed it now. Thanks for the issue.

Greetings Damien
